app/models/groupe_instructeur.rb

```diff
@@ -59,5 +59,15 @@ def can_delete?
 
   def toggle_routing
     procedure.update!(routing_enabled: procedure.groupe_instructeurs.active.many?)
+
+    if procedure.groupe_instructeurs.active.size == 2 && !procedure.has_a_tdc_routage?
```

Comment:
```suggestion
    if procedure.groupe_instructeurs.active.many? && !procedure.has_a_tdc_routage?
```

pour être cohérent avec la ligne 61

Reply:
je crois que là il faut laisser tel quel, parce qu'on veut ajouter le type de champ routage seulement quand on pase de un à deux groupes actifs